gdal/ogr/ogrsf_frmts/flatgeobuf/flatbuffers/code_generators.h

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
is it a generated file ?

Reply:
Also part of include files from the flatbuffers project.